src/components/composer/composer/composer.ts

```diff
@@ -568,27 +569,31 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.functionDescriptionState.showDescription = false;
 
     if (content.startsWith("=")) {
-      const tokenAtCursor = this.env.model.getters.getTokenAtCursor();
-      if (tokenAtCursor) {
-        const { xc } = splitReference(tokenAtCursor.value);
-        if (
-          tokenAtCursor.type === "FUNCTION" ||
-          (tokenAtCursor.type === "SYMBOL" && !rangeReference.test(xc))
-        ) {
-          // initialize Autocomplete Dropdown
-          this.showAutocomplete(tokenAtCursor.value);
-        } else if (tokenAtCursor.functionContext && tokenAtCursor.type !== "UNKNOWN") {
-          // initialize Formula Assistant
-          const tokenContext = tokenAtCursor.functionContext;
-          const parentFunction = tokenContext.parent.toUpperCase();
-          const description = functions[parentFunction];
-          const argPosition = tokenContext.argPosition;
-
-          this.functionDescriptionState.functionName = parentFunction;
-          this.functionDescriptionState.functionDescription = description;
-          this.functionDescriptionState.argToFocus = description.getArgToFocus(argPosition + 1) - 1;
-          this.functionDescriptionState.showDescription = true;
-        }
+      const token = this.env.model.getters.getTokenAtCursor();
```

Comment:
Not related to this PR, but a range token `NotARealSheet!A1:B2` have no color (not a valide range) but still have an highlight